packages/create-plugin/templates/common/docker-compose.yaml

```diff
@@ -6,6 +6,7 @@ services:
     build:
       context: ./.config
       args:
+        grafana_image: ${GRAFANA_IMAGE:-{{~grafanaImage~}} }
```

Comment:
Nice! We probably need to [update the docs](https://grafana.github.io/plugin-tools/docs/docker#configure-the-grafana-image) for this. WDYT?

Reply:
not sure we need, it seems still to be correct?